enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -1144,10 +1144,13 @@ func (s *SchedulerServer) claimTask(ctx context.Context, taskID string, claimTim
 		// Success
 		break
 	case 10:
+		log.CtxErrorf(ctx, "claimTask %q error: task does not exist", taskID)
```

Comment:
Looking at executor logs, these are extremely common too -- maybe we can log these at Info level?

```suggestion
		log.CtxInfof(ctx, "claimTask %q error: task does not exist", taskID)
```